File: python/isprime.py
#!/usr/bin/env python3
#Is number prime?

#Isprime... return true only if argument number is prime
def is_prime(number):
    if number < 2:
        return False
    for i in range(2,number+1):
        if   i != number:
            if number % i == 0 :
                return False
    return True

# define a range to check prime number
def print_prime_number(ran):
    for i in range(2,ran+1):
        if is_prime(i):
            print(i)

print_prime_number(100)

File: go/isprime.go
// package main Is a prime number?
package main

import "fmt"

// Isprime... return true only if argument number is prime
func Isprime(number int) bool {
	for i := 2; i <= number; i++ {
		if i != number {
			if number%i == 0 {
				return false
			}
		}

	}
	return true
}

// PrintPrimeNUmber show in a range  all prime number
func PrintPrimeNumber(ran int) {
	for i := 2; i <= ran; i++ {
		if Isprime(i) {
			fmt.Printf("%v ", i)
		}

	}
	fmt.Println()
}
func main() {
	PrintPrimeNumber(100)

}
